packages/teams-js/src/public/meeting.ts

```diff
@@ -560,6 +570,10 @@ export namespace meeting {
     );
   }
 
+  /**
+   * @hidden
```

Comment:
Why are they tagged @hidden ?
It will be helpful for the SDK developers to know what the helper function input/output.

Reply:
The @hidden tag is so the docs don't show up in the public documentation for the clients rather than for the SDK devs.